### PR TITLE
pods: Dashboard is the default pod-detail tab

### PR DESCRIPTION
**Pull request by mareklibra**
Depends on:
- [x] #2796 

Just the last commit is relevant.

I would postpone merging of this PR till more cards land on the Pod Detail Dashboard page.

### CODE REVIEW

**Comment by mareklibra** [comment]
Cc @andybraren

**Comment by mareklibra** [comment]
Rebased, ready for review.
But let's hold merging of this till more pod-dashboard cards lands.

File: frontend/public/components/pod.tsx [hunk 1]
```diff
@@ -405,11 +406,11 @@ export const PodsDetailsPage: React.FC<PodDetailsPageProps> = (props) => (
     menuActions={menuActions}
     pages={[
       {
-        href: 'dashboard', // TODO: make it default once additional Cards are implemented
+        href: POD_DETAIL_DASHBOARD_HREF,
```
Comment (vojtechszocs) [inline frontend/public/components/pod.tsx]: I'd suggest to stick with existing convention

```ts
href: '',
```

which makes it clear that this is the default landing page/tab within the details page.

Having a constant for an empty string adds one more level of indirection.

Comment (mareklibra) [reply frontend/public/components/pod.tsx]: done

**Comment by vojtechszocs** [comment]
@mareklibra LGTM - please un-hold when ready.

**Comment by andybraren** [comment]
Agreed, please keep the hold for now, I'm working on confirming the path forward for this dashboard. (All other dashboards are still good to proceed)

**Comment by mareklibra** [comment]
The pod-dashboard has been descoped for 4.2.

**Pull request closed without merging**